tests/strategies/test_ddp_strategy.py

```diff
@@ -119,11 +119,29 @@ def test_ddp_configure_ddp():
         max_epochs=1,
         strategy=ddp_strategy,
     )
-    # test do not wrap the model if trainerFN is not fitting
+    # test do not wrap the model if TrainerFn is not fitting
     trainer.state.fn = TrainerFn.VALIDATING
     trainer.strategy.connect(model)
     trainer.lightning_module.trainer = trainer
     trainer.strategy.setup_environment()
     trainer.strategy.setup(trainer)
     # in DDPStrategy configure_ddp(), model are still LightningModule
     assert isinstance(trainer.model, LightningModule)
+
+
+@RunIf(min_gpus=1)
+def test_ddp_sync_batchnorm():
```

Comment:
this test would also pass on master, right? 
You would need to set TrainerFn to something other than FITTING to validate that the model is not wrapped. 
The test would then fail, because in ddp we still wrap the model regardless of FITTING or not. I fixed this in a previous commit but my commit got removed (see previous comment), not sure why but I'm pretty confident it was right.